src/Fynance/YahooTicker.cs

```diff
@@ -80,7 +80,9 @@ public override async Task<FyResult> GetAsync()
 
 		private async Task<HttpResponseMessage> GetResponse(string url)
 		{
-			using (var http = new HttpClient())
+			var handler = new HttpClientHandler();
+			handler.UseProxy = false;
+			using (var http = new HttpClient(handler))
```

Comment:
haven't changed it as it shouldn't be in that PR but usually it's bad practice to dispose HttpClient and reuse the same one all the time.
Might be worth adding http client to a constructor and let the user add their own httpClient if they wish 
and continue to use that way if they don't provide it

Reply:
Looks good Erwan! 
I think would be nice to encapsulate the http client instance and add a second constructor which can take the instance of httpClient. Maybe add a method `SetClient(httpClient)`. When it is not defined, initialize it with a default instance.
What do you think?

Reply:
Yes I think that would be a good thing to do. Might be worth to do it in another PR

Reply:
Cool, I will close this one, send a new one when you are done and thank you for your contribution!

Reply:
I mean this one can be merged in the current state. That's why I believe the other optimisation can be done in another PR

Reply:
That's fine, send a new PR with all the modifications around the http client and let's see.